browsergym/experiments/src/browsergym/experiments/loop.py

```diff
@@ -657,6 +657,94 @@ def summary_info(self) -> dict:
                 self._summary_info = json.load(f)
         return self._summary_info
 
+    @property
+    def tape(self) -> dict:
+        """
+        TapeAgents (https://github.com/ServiceNow/TapeAgents) framework compatibility.
+        Exports experiment trace in the format of serialized tape.
+        Reuses tape segments if they were already placed in the agent_info during the experiment.
+
+        :returns: dict: serialized tape of the experiment
+        """
+        steps = []
+        for step_info in self.steps_info:
+            if "tape_segment" in step_info.agent_info["extra_info"]:
+                tape_segment = step_info.agent_info["extra_info"]["tape_segment"]
+            else:
+                tape_segment = self._create_tape_segment(step_info)
+            steps += tape_segment
+        metadata = dict(
+            id=str(uuid.uuid4()),
+            author="browsergym_agent",
```

Comment:
Maybe the agent name could be re-used here?
```suggestion
            author=f"browsergym_agent_[{agent_name}]",
```